node/src/chain_spec.rs

```diff
@@ -169,6 +192,101 @@ pub fn foucoco_config() -> FoucocoChainSpec {
 				sudo_account.clone(),
 				foucoco::PARACHAIN_ID.into(),
 				false,
+				vec![],
+			)
+		},
+		// Bootnodes
+		Vec::new(),
+		// Telemetry
+		None,
+		// Protocol ID
+		Some("foucoco"),
+		// Fork ID
+		None,
+		// Properties
+		Some(properties),
+		// Extensions
+		ParachainExtensions {
+			relay_chain: "kusama".into(), // You MUST set this to the correct network!
+			para_id: foucoco::PARACHAIN_ID,
+		},
+	)
+}
+
+pub fn foucoco_standalone_config() -> FoucocoChainSpec {
```

Comment:
You think a different name will be better after? Since we where talking about the naming also [here](https://github.com/pendulum-chain/pendulum/pull/462#discussion_r1611445952).

Reply:
No no, nevermind. It's like the standalone chain after all.